content/05.main-text.md

```diff
@@ -347,15 +347,27 @@ Consequently, degree-grouping excels where it is needed most.
 One final benefit of degree-grouping is that reduces the disk space required to store null DWPC summary statistics.
 For example, with 20,945 genes in Hetionet v1.0, there exists 438,693,025 gene pairs.
 Gene nodes have 302 distinct degrees for _interacts_ edges, resulting in 91,204 degree pairs.
-This equates to an 4810-fold reduction in the number of summary statistics that need to be stored to represent the null DWPC distribution for a metapath starting and ending with a _Gene--interacts--Gene_ metaedge.
+This equates to an 4,810-fold reduction in the number of summary statistics that need to be stored to represent the null DWPC distribution for a metapath starting and ending with a _Gene--interacts--Gene_ metaedge.
 
-<!--
-TODO: somewhere link to notebook https://github.com/greenelab/hetmech/blob/042063fb559048e52b3dc2731b6d6c6836f698cf/explore/bulk-pipeline/bulk.ipynb
+We store the following null DWPC summary statistics for each metapath--source-degree--target-degree combination:
+total number of null DWPCs,
+total number of nonzero null DWPCs,
+sum of null DWPCs,
+sum of squaured null DWPCs,
+and number of permuted hetnets.
+These values are sufficient to estimate the _p_-value for a DWPC, by either fitting a gamma-hurdle null distribution or generating an empiric _p_-value.
+Furthermore, these statistics are additive accross permuted hetners.
+Their values are always a running total and can be updated incrementally as statistics from each additional permuted hetnet become available.
```

Comment:
@zietzm writing this section, I am still dumbfounded by the design of the null DWPC computation. So many hurdles we had to overcome to make the task tractable!